src/components/inline/InputText/InputText.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./InputText.module.css";
+
+export const InputTextPropType = PropTypes.shape({
+  placeholderText: PropTypes.string,
+  widthDesktop: PropTypes.string.isRequired,
+  widthMobile: PropTypes.string.isRequired,
```

Comment:
I think this probably makes the component too awkward to style and control, since you are forced to set the width for both desktop and mobile in the JSX code. What if you just have the input element take up 100% of the width of its parent, and then allow a user to wrap this element with something else if they want to constrain the width? For example:

```jsx
<span className="my-wrapper"><InputText></span>
```

```css
.my-wrapper {
  width: 400px;
}

@media (max-width: 720px) {
  .my-wrapper {
    width: 200px;
  }
}
```

Note that if you set `width: 100%`, you'll need to set `box-sizing: border-box` (or alternatively, you'll have to use some `calc()`s to compensate for the border and padding sizes).

Reply:
ok

Reply:
ok